tests/bluetooth/controller/common/src/helper_pdu.c

```diff
@@ -1,557 +0,0 @@
-/*
```

Comment:
Why is this file deleted in this commit?

Reply:
This seems to be an unsuccessful attempt to move a file from one commit to another.
Will be fixed.